Source/Events.Store/Event.cs

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Dolittle.Artifacts;
+
+namespace Dolittle.Runtime.Events.Store
+{
+    /// <summary>
+    /// Abstract type for Committed/UncommittedEvent.
```

Comment:
For abstract classes we usually say that it is a 'basis' for something.
```suggestion
    /// Represents the basis for an event.
```